lib/disability_compensation/providers/claims_service/evss_claims_service_provider.rb

```diff
@@ -6,8 +6,8 @@
 
 class EvssClaimsServiceProvider
   include ClaimsServiceProvider
-  def initialize(current_user = nil, auth_headers = nil)
-    @service = EVSS::ClaimsService.new(current_user, auth_headers)
+  def initialize(auth_headers)
+    @service = EVSS::ClaimsService.new(auth_headers)
```

Comment:
the base class of the evss claims service provider for the evss implementation does not utilize `current_user`